types/groups/groupmembers.d.ts

```diff
@@ -13,32 +13,33 @@ declare module 'libsession_util_nodejs' {
   };
 
   type MemberStateGroupV2 =
-    | 'INVITE_NOT_SENT' // as soon as we've scheduled that guy to be invited, but before we've tried sending the invite message
-    | 'INVITE_FAILED'
-    | 'INVITE_SENT'
+    | 'INVITE_UNKNOWN' // fallback case
```

Comment:
Maybe use docstrings so we can see the description in session-desktop?

Reply:
Added on docstring on the type itself as docstring do not work on separate strings of a union. This looks ok to me 
![image](https://github.com/user-attachments/assets/38c641ab-635a-4fa1-999f-c28e77def41f)
What do you think?

Reply:
Yeah fair enough. that is annoying that it is not supported.